packages/block-library/src/tag-cloud/edit.js

```diff
@@ -168,7 +168,7 @@ function TagCloudEdit( { attributes, setAttributes, taxonomies } ) {
 			{ inspectorControls }
 			<div { ...useBlockProps() }>
 				<ServerSideRender
-					key="tag-cloud"
```

Comment:
I found that only this block has a `key` prop among the blocks using the `ServerSideRender` component.
But since the `key` prop is not accepted by the `ServerSideRender` component, I removed it.